Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java

```diff
@@ -40,6 +40,7 @@
     private static final long serialVersionUID = 1L;
     private static final Logger logger = Logger.getLogger(Installer.class.getName());
     private static Installer instance;
+    public static boolean enableTimeline = true;
```

Comment:
Our coding guidelines prohibit global variables. Please put this value in a properties file for Timeline to read. There is a ModuleSettings utility class that could be used for managing the properties file.

Note that you will need to reset the property to enabled at the beginning of this set LAF operation.